pkg/runtime/runtime.go

```diff
@@ -76,17 +74,6 @@ type Container struct {
 	VolumeMounts []corev1ac.VolumeMountApplyConfiguration
 }
 
```

Comment:
Are we going to refactor the CountForNonTrainer in the next PRs ?

Reply:
As we discussed today, I will open 

1. Remove Info.Trainer <- This PR
2. Remove podSpecReplicas (runtime package)
3. Obtain TrainingRuntime Applyconfiguration and reuse it in a whole of KF Pipeline Framework
4. Change the parameter propagation order between TrainJob and Runtime and PodSet